service/src/test/java/bio/terra/workspace/service/resource/controlled/flight/clone/bucket/CopyGcsBucketDefinitionStepTest.java

```diff
@@ -47,18 +62,67 @@
 public class CopyGcsBucketDefinitionStepTest extends BaseUnitTestMockGcpCloudContextService {
   private CopyGcsBucketDefinitionStep copyGcsBucketDefinitionStep;
 
+  @Autowired private WorkspaceDao workspaceDao;
+  @Autowired private ControlledResourceService controlledResourceService;
+
   @Mock FlightContext mockFlightContext;
+  @Mock private StorageCow mockStorageCow;
+  @Mock private BucketCow mockBucketCow;
+  @Mock private Storage mockStorageClient;
+  @Mock private Storage.Buckets mockBuckets;
+  @Mock private Storage.Buckets.Get mockStorageBucketsGet;
+  @Mock private GcpCloudContext mockGcpCloudContext;
+  @Mock private Policy mockPolicy;
+
+  @Captor private ArgumentCaptor<BucketInfo> bucketInfoCaptor;
```

Comment:
were you planning to use this? i didn't see you assert the captured value below.